app/services/photos.js

```diff
@@ -16,3 +17,27 @@ exports.getPhotosForAlbum = async albumId => {
     throw errors.albumApiError(err.message);
   }
 };
+
+exports.getPhotosForAlbum = async albumId => {
+  try {
+    return await getPhotosForOneAlbum(albumId);
```

Comment:
you dont need to await it if you are returning it, it will be a promise either way